network/src/sync/blocks.rs

```diff
@@ -97,7 +83,7 @@ impl<S: Storage + Send + std::marker::Sync + 'static> Node<S> {
         block: Vec<u8>,
         is_block_new: bool,
     ) -> Result<(), NetworkError> {
-        let block_struct = match Block::deserialize(&block) {
+        let block_struct = match Block::<Transaction<Components>>::deserialize(&block) {
```

Comment:
We could introduce a type alias for `Transaction<Components>` to make these calls more compact.

Reply:
we are going to remove this stuff once we have proper serialization and such. stop-gap.